### DISCUSSION TITLE
Generate SVG Without "Chiral" Label

### DISCUSSION THREAD

**Discussion by Chang254** [epam/ketcher#4655] [General]
Hello,

I am trying to generate an SVG from the editor without the "Chiral Label".  Is there a way to do this?  

Here is the drawn molecule in the editor with labels off for the stereocenter:
![Screenshot 2024-05-18 at 4 46 27 PM](https://github.com/epam/ketcher/assets/35904733/f3146051-7a58-461c-b76f-c372637a0bdb)

When I use generateImage or save as SVG as shown below, I want a way to remove the "Chiral" text at the top:
![Screenshot 2024-05-18 at 4 46 54 PM](https://github.com/epam/ketcher/assets/35904733/1928ee5f-0499-47d4-8b14-9960872ef6a2)

**Comment by capichou** [comment]
Hello,

It's been quite a long time... Is there any solution now? Sure, I can edit it with inkscape, but I don't even think "Chiral" should display by default.

**Comment by rrodionov91** [comment]
Hi @capichou 

We have such issue in our backlog https://github.com/epam/ketcher/issues/5570
It is not planned to any release.
I initiated discussion with team. We will discuss and let you know about possibilities.
